kinds/dashboard/dashboard_kind.cue

```diff
@@ -447,11 +445,11 @@ lineage: schemas: [{
 			disabled?: bool
 			// Optional frame matcher. When missing it will be applied to all results
 			filter?: #MatcherConfig
+			// Where to pull DataFrames from as input to transformation
+			topic?: "series" | "annotations" // replaced with common.DataTopic
```

Comment:
a large defect of this system is that you can not reference shared types from common 🤦🏻 

As a temporary solution we duplicate the definition here, and replace the type in the veneer